### PR TITLE
Fixed item scripts

### PR DESCRIPTION
**Pull request by zackdreaver**
* **Addressed Issue(s)**: N/A

* **Server Mode**: Renewal

* **Description of Pull Request**: Fixed and added some items

### CODE REVIEW

File: db/re/item_db.txt [hunk 1]
```diff
@@ -11318,8 +11318,9 @@
 //
 23012,S_Small_Mana_Potion,[Sale] Small Mana Potion,2,0,,10,,,,,0xFFFFFFFF,63,2,,,,,,{ bonus_script "bonus2 bRegenPercentSP,5,5000;",600,9; /* fix me: unknown flag and specialeffect ; disabled when LK_BERSERK */ },{},{}
 23016,Cursed_Fragment,Cursed Fragment,2,1,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ callfunc("F_Cursed_Fragment"); },{},{}
-23038,S_Slim_White_Box,[Sale] Slim White Potion Box,2,0,,0,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 547,200; },{},{}
-23043,S_Seed_Of_Yggdrasil_Box,[Sale] Yggdrasil Seed Box,2,0,,0,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 608,30; },{},{}
+23038,S_Slim_White_Box,[Sale] Slim White Potion Box,2,0,,0,,,,,0xFFFFFFFF,63,2,,,,,,{ getitem 11573,200; },{},{}
+23042,S_Seed_Of_Yggdrasil,Yggdrasil Seed,0,0,,300,,,,,0xFFFFFFFF,63,2,,,,,,{ percentheal 50,50; },{},{}
```
Comment (aleos89) [inline db/re/item_db.txt]: Might be worthwhile to add the cooldown for this as well?
`재사용 대기시간 : 3초`

File: db/re/item_db.txt [hunk 2]
```diff
@@ -877,7 +877,7 @@
 1525,Long_Mace,Long Mace,5,20,,800,135,,3,0,0x00008110,63,2,2,4,40,1,8,{ bonus bLongAtkDef,10; },{},{}
 1526,Slash,Slash,5,20,,1000,145,,1,0,0x00008110,63,2,2,4,40,1,8,{ bonus2 bAddRace,RC_Undead,15; bonus2 bComaRace,RC_Undead,10; bonus2 bExpAddRace,RC_Undead,5; },{},{}
 1527,Quadrille,Quadrille,5,20,,900,165,,1,0,0x00008110,63,2,2,4,40,1,8,{ bonus2 bAddRace,RC_Undead,10; bonus2 bAddRace,RC_DemiHuman,10; bonus2 bAddRace,RC_Player,10; bonus2 bAddEle,Ele_Earth,10; },{},{}
-1528,Grand_Cross,Grand Cross,5,20,,1500,140,,1,0,0x00008110,63,2,2,4,40,1,8,{ bonus bAtkEle,Ele_Holy; skill "PR_TURNUNDEAD",3; bonus3 bAutoSpell,"PR_TURNUNDEAD",3,100; bonus2 bSPDrainValueRace,RC_Undead,1; bonus2 bSPGainRace,RC_Undead,3; },{},{}
```
[the remark this answers is not in the document]
Comment (zackdreaver) [reply db/re/item_db.txt]: What list?
Auto spell skill shouldn't listed on skill window, or am I missing something?

[the remark this answers is not in the document]
Comment (Daegaladh) [reply db/re/item_db.txt]: The item grants both autospell and the skill at level 3.

[the remark this answers is not in the document]
Comment (aleos89) [reply db/re/item_db.txt]: We confirmed it to be a pre-renewal only bonus. So this change is good.

[the remark this answers is not in the document]
Comment (zackdreaver) [reply db/re/item_db.txt]: Ok

**Comment by zackdreaver** [comment]
Ok, it's done.
Unless there is another request for change.

**Pull request merged**